### PR TITLE
solution

### PR DESCRIPTION
**Pull request by SerjDSA**
[DEMO LINK](https://serjdsa.github.io/js_employees_table_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,218 @@
...
+
+  cell.innerHTML = `<input class='cell-input' value='${editedCell}' />`;
+
+  const input = e.target.querySelector('.cell-input');
+
+  input.focus();
+
+  // eslint-disable-next-line no-shadow
+  const saveChanges = (e) => {
+    if (e.type === 'blur' || (e.type === 'keyup' && e.key === 'Enter')) {
```
Comment (TarasHoliuk) [inline src/scripts/main.js:208]: Would be nice to properly format salary after editing (you can move it to some function and reuse when editing and adding new row)

image

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,218 @@
...
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const rows = [...tbody.rows];
+
+let sortedColumn = null;
+let sortOrder = 'asc';
+
+tableHead.addEventListener('click', (e) => {
```
Comment (TarasHoliuk) [inline src/scripts/main.js:10]: after sort all previously added rows disappear - should be fixed

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Hey! GJ 👍 
But a few fixes are required

**Review by SerjDSA — COMMENTED** [review]
[DEMO LINK](https://serjdsa.github.io/js_employees_table_DOM/)

**Pull request closed without merging**